site/_docs/reference.md

```diff
@@ -2108,6 +2109,7 @@ semantics.
 | o | GREATEST(expr [, expr ]*)                      | Returns the greatest of the expressions
 | o | LEAST(expr [, expr ]* )                        | Returns the least of the expressions
 | o p | TRANSLATE(expr, fromString, toString)          | Returns *expr* with all occurrences of each character in *fromString* replaced by its corresponding character in *toString*. Characters in *expr* that are not in *fromString* are not replaced
+| m | REVERSE(string)                                  | Returns the reverse order of string values
 
```

Comment:
Move this item together with other mysql functions.

Reply:
done.